howtoread/index.md: explain scope of the handbook (#62)
void-linux/void-docs: src/howtoread/index.md

# How To Read The Handbook

This handbook is not an extensive guide on how to use and configure common Linux
software. The purpose of this document is to explain how to install, configure,

> and maintain Void Linux systems, including the differences from other common

Reviewer: `Focusing on`, perhaps?

> and maintain Void Linux systems, focusing on the differences from other common

Reviewer 2: ```suggestion
and maintain Void Linux systems, and to explain differences between common Linux distributions and Void.
```

The wording of this seemed a little off.  Maybe explain is the wrong word too. Highlight? Cover? Something like that. I think what I'm trying to get at is that it's not that this guide focuses on the differences, just that there will be differences and this guide documents them.

Author: I did feel iffy about the wording here. I will come up with something better. I always appreciate your reviews and this is very helpful :)

Linux distributions.

Those looking for tips and tricks on how to configure a Linux system in general
should consult upstream software documentation. Additionally, the [Arch
Wiki](https://wiki.archlinux.org/) provides a fairly comprehensive outline of
common Linux software configuration, as well as the

> [google](https://www.google.com) search engine.

Reviewer: `and a variety of internet search engines exist for further assistance` maybe.
